app/services/get_constituency.rb

```diff
@@ -11,6 +11,7 @@ def initialize(postcode, client)
   end
 
   def get
+    return nil if @postcode.nil? || @postcode.empty?
```

Comment:
@yeseniamolinab nil/empty postcode was the issue after all. Merging.

Reply:
Thanks, Omar!